parser/src/parser/tests/arithmetic_ops.rs

```diff
@@ -12,17 +12,15 @@ fn single_addition() {
     let source = "
     transition_constraints:
         enf clk' + clk = 0";
-    let expected = Source(vec![SourceSection::TransitionConstraints(
-        TransitionConstraints {
-            transition_constraints: vec![TransitionConstraint::new(
-                TransitionExpr::Add(
-                    Box::new(TransitionExpr::Next(Identifier("clk".to_string()))),
-                    Box::new(TransitionExpr::Elem(Identifier("clk".to_string()))),
-                ),
-                TransitionExpr::Const(0),
-            )],
-        },
-    )]);
+    let expected = Source(vec![SourceSection::TransitionConstraints(vec![
+        TransitionStmt::Constraint(TransitionConstraint::new(
+            TransitionExpr::Add(
+                Box::new(TransitionExpr::Next(Identifier("clk".to_string()))),
+                Box::new(TransitionExpr::Elem(Identifier("clk".to_string()))),
+            ),
+            TransitionExpr::Const(0),
+        )),
+    ])]);
```

Comment:
Probably not for this PR, but we can make things like this much more readable by importing enum variants (e.g., `use TransitionStmt::*;`, `use SourceSection::*;` etc. For example, this could look something like:
```Rust
let expected = Source(vec![TransitionConstraints(vec![
    Constraint(TransitionConstraint::new(
        Add(
            Box::new(Next(Identifier("clk".to_string()))),
            Box::new(Elem(Identifier("clk".to_string()))),
        ),
        Const(0),
    )),
])]);
```